src/java/map/FloatMask.java

```diff
@@ -907,6 +907,10 @@ public String toHash() throws NoSuchAlgorithmException {
     }
 
 
+    public FloatMask startVisualDebugger() {
+        return startVisualDebugger(toString(), Util.getStackTraceParentClass());
+    }
+
```

Comment:
Why was this added the default toString is not that meaningful

Reply:
It has some relevant information and is easier/faster for me to use than having to manually write a name each time I want to visually debug something.